easybuild/easyconfigs/s/spaCy/spaCy-3.7.4-foss-2023a.eb

```diff
@@ -0,0 +1,89 @@
+easyblock = 'PythonBundle'
+
+name = 'spaCy'
+version = '3.7.4'
+
+homepage = 'https://spacy.io/'
+description = "Industrial-strength Natural Language Processing (NLP) in Python."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('poetry', '1.7.1'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('tqdm', '4.66.1'),
+    ('pydantic', '2.5.3'),
+    ('hatchling', '1.18.0')
```

Comment:
`hatchling` should most probably be a builddep instead of a dep

Reply:
I thought so too but see line 484 https://gist.github.com/lcniel/c44522f699832ce8cfb28b110c7d5bf6:

`hatch-fancy-pypi-readme 24.1.0 requires hatchling, which is not installed.`

Reply:
It might be a bit painful to have hatchling as  runtime dep, it would collide with anything else which might pull different versions (as it's typically ok for something that's just a build dep).

In this case:
hatch-fancy-pypi-readme 24.1.0 requires hatchling, which is not installed.
this is an indirect dep for something that looks like a hatchling.

Either we split out hatch-fancy-pypi-readme as it's own package (like hatch-jupyter-builder-0.9.1-GCCcore-13.2.0.eb) and only use that as a builddep, or just skip this completely as it really does not seem necessary for the build. Making some fancy readme for pypi i guess? Nothing we will ever see of notice.

Reply:
I will try to patch it out.

Reply:
Seems like it just works now without any added patching needed, just taking out the dep. Weird, will test more